Example/Example/SeriesWireframe.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ComicsWireframe.swift
+//  Example
+//
+//  Created by Davide Mendolia on 03/12/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+
+class SeriesWireframe: ExampleWireframe {
+
+    func presentSerieDetailViewController(serieName: String) {
+        let viewController = serviceLocator.provideSerieDetailViewController()
```

Comment:
I'm already working on it, the name of the branch is ``marvel-ui-example-serie-detail``. I've pushed my first commit in that branch with that change ;)

![Party](https://media3.giphy.com/media/hmYq2fIb9u3lu/200_s.gif)